internal/sirius/case_type.go

```diff
@@ -11,7 +11,7 @@ const (
 
 func ParseCaseType(s string) (CaseType, error) {
 	switch s {
-	case "lpa":
+	case "lpa", "LPA":
```

Comment:
do you need this, and if so will you need `EPA` for the other?

Reply:
Yeah the api request returns the case type in upper case form, it will be needed for epa